### DISCUSSION TITLE
Change ports on Single Server Installation

### DISCUSSION THREAD

**Discussion by Mozitux** [mediacms-io/mediacms#499] [Q&A]
Hello;

I use port forwarding and would like to know how I can change MediaCMS ports 80 and 443. I found this post (https://github.com/mediacms-io/mediacms/discussions/304), but it's about Docker, which I'm not.

I'm using the latest version of MediaCMS, how can I change the ports?

**Comment by mgogoulos** [comment]
Pinging @swiftugandan to this, if you have some ideas!

**Reply by Mozitux** [reply]
Hey, I've found how to change the ports:

Go to:  `/etc/nginx/sites-available/`
Open `mediacms.io`
Change the ports numbers and restart nginx with `service nginx restart`
